hysds/es_util.py

```diff
@@ -51,5 +57,9 @@ def get_grq_es():
                 retry_on_timeout=True,
             )
         else:
-            GRQ_ES = ElasticsearchUtility(es_url, logger)
+            if es_cluster_mode:
+                hosts = [app.conf.JOBS_ES_URL, app.conf.GRQ_ES_URL, app.conf.METRICS_ES_URL]
+            else:
+                hosts = [app.conf.JOBS_ES_URL]
```

Comment:
Should this fallback to what it was before?

```suggestion
                hosts = [es_url]
```